client/src/components/UIPatterns/UIPatternsSidebar/UIPatternsSidebar.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Sidebar } from '_common';
+
+function UIPatternsSidebar() {
+  const sidebarItems = [
+    { to: 'history', iconName: 'trash', children: 'History' },
+    { to: 'allocations', iconName: 'alert', children: 'Allocations' },
+    {
+      to: 'applications',
+      iconName: 'alert',
+      children: 'Disabled',
+      disabled: true,
+    },
+    { to: 'ui-patterns', iconName: 'file', children: 'Here' },
```

Comment:
__Concern__
Can a `sidebarItems` object set a component as `children`?

<details><summary>Example</summary>

```js
    const Thing = <Icon name="alert" />;
    const sidebarItems = [
        // ...
        { to: 'ui-patterns', iconName: 'file', children: Thing },
    ]
```

(My syntax may be wrong.)

</details>

<details><summary>Goal</summary>

Convert all the Sidebars in the app into to use this component, even [the main app Sidebar which includes a `<HistoryBadge>` in of its item's children](https://github.com/TACC/Core-Portal/blob/272ab4f/client/src/components/Sidebar/Sidebar.jsx#L101). _The others ([Allocations](https://github.com/TACC/Core-Portal/blob/272ab4f/client/src/components/Allocations/AllocationsLayout.jsx#L47-L48), [DataFilesSidebar](https://github.com/TACC/Core-Portal/blob/27c8d95/client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.jsx#L122-L123), [History](https://github.com/TACC/Core-Portal/blob/0eb07ac/client/src/components/History/History.js#L51-L52)) all just icons and labels, so they should be fine._